contrib/test/crio-integration-playbook.yaml

```diff
@@ -27,28 +82,57 @@
       name: "{{ item }}"
       state: latest
     with_items:
-      - wget
-      - git
-      - make
-      - gcc
-      - tar
-      - libseccomp-devel
-      - golang
-      - glib2-devel
-      - glibc-static
-      - container-selinux
       - btrfs-progs-devel
+      - container-selinux
+      - skopeo-containers
+      - curl
       - device-mapper-devel
-      - ostree-devel
+      - expect
+      - findutils
+      - gcc
+      - git
+      - glib2-devel
       - glibc-devel
+      - glibc-static
+      - golang
       - gpgme-devel
+      - hostname
+      - iproute
+      - iptables
+      - krb5-workstation
       - libassuan-devel
+      - libffi-devel
       - libgpg-error-devel
-      - pkgconfig
-      - skopeo-containers
-      - oci-systemd-hook
+      - libguestfs-tools
+      - libseccomp-devel
+      - libvirt-client
+      - libvirt-python
+      - libxml2-devel
+      - libxslt-devel
+      - make
+      - mlocate
+      - nfs-utils
+      - nmap-ncat
+      - npm
       - oci-register-machine
+      - oci-systemd-hook
       - oci-umount
+      - openssl
+      - openssl-devel
+      - ostree-devel
+      - pkgconfig
+      - python
+      - python2-boto
+      - python2-crypto
+      - python2-mock
+      - python-click
+      - python-devel
+      - python-virtualenv
+      - PyYAML
+      - redhat-rpm-config
+      - rpcbind
+      - rsync
+      - sed
       - socat
```

Comment:
There are probably a few deps. above that are not strictly needed.  Leaving that for someone else to fix 😀